backend/clubs/serializers.py

```diff
@@ -600,9 +600,11 @@ def get_email(self, obj):
     def get_image(self, obj):
         if not obj.public:
             return None
-        if not obj.person.profile.image:
-            return None
-        image_url = obj.person.profile.image.url
+        if not obj.image:
+            if not obj.person.profile.image:
+                return None
```

Comment:
This could be reduced to a single if statement.

Reply:
Lucy mentioned this earlier I think!

Reply:
Changed!